controllers/articles.controller.js

```diff
@@ -21,4 +21,18 @@ const getArticlesId = (req, res, next) => {
         next(err);
       });
   };
-module.exports = {getArticlesId, getArticles}
\ No newline at end of file
+  const patchArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    const { inc_votes } = req.body;
+    if (inc_votes === undefined || typeof inc_votes !== 'number') {
```

Comment:
If inc_votes is not a number, PSQL should throw an error, so you shouldn't have to deal with it here 👍